### DISCUSSION TITLE
Voting for Development Factor Selections

### DISCUSSION THREAD

**Discussion by Obalus2022** [casact/chainladder-python#323] [Q&A]
Hi I have a question on the development factor functions.  Once you determine development factors is there any way to combine different Development factors, basically what I want to emulate is an average I use which takes the average of industry LDFs and ldfs which I create from triangles.  Is that sort of functionality available?

Thanks in advance.  THis is a tremendous resource.

**Comment by kennethshsu** [comment]
I don't think that's currently possible, you would have to use something like [DevelopmentConstant(...)](https://chainladder-python.readthedocs.io/en/latest/development.html#developmentconstant).
